src/sass/components/_customer-reviews.scss

```diff
@@ -91,27 +100,150 @@
   }
 
   &__navigation {
-    margin-top: 3rem;
+    margin-top: 30px;
     display: flex;
     justify-content: center;
-    gap: 1.8rem;
+    gap: 18px;
   }
 
   &__icon-home {
     fill: $blueberry-lemonade;
   }
 
   &__nav-sec-rev {
-    width: 1.2rem;
-    height: 1.2rem;
+    width: 12px;
+    height: 12px;
     border-radius: 50%;
     background-color: $bgreview-grey;
   }
 
   &__nav-third-rev {
     border-radius: 50%;
-    width: 1.2rem;
-    height: 1.2rem;
+    width: 12px;
+    height: 12px;
     background-color: $bgreview-grey;
   }
 }
+
+//tablet
+@media screen and (min-width: 768px) {
+  .container-reviews {
+    max-width: 768px;
+    margin: 0 74px;
+
+    &__text-block {
+      position: relative;
```

Comment:
nie nadpisujemy niepotrzebnie styli, wystarczy że dla każdego elementu będziesz nadpisywał tylko te style które się zmieniają, czyli tutaj to position: relative masz takie samo dla każdej wielkości ekranu, więc jest niepotrzebnie nadpisane. niżej widzę że jest np background-color, w pseudo-elemencie position: absolute, background-image, content, opacity itd